sdk/go/common/resource/config/object.go

```diff
@@ -115,6 +116,8 @@ func (c object) decrypt(ctx context.Context, path resource.PropertyPath, decrypt
 			vs[k] = pv
 		}
 		return NewPlaintext(vs), nil
+	case nil:
+		return Plaintext{}, nil
```

Comment:
Can you add a test that exercises this part of the code?

Reply:
There is.  https://github.com/pulumi/pulumi/pull/14149/files#diff-2ee802e4d2ddbf1ae1787d72e96c45f45bfa2586c4dab5dc9f663d1705f7bc66R40-R41 should exercise this, and panics if we're missing this piece here.